exonum/src/lib.rs

```diff
@@ -18,6 +18,26 @@
 
 #![deny(missing_debug_implementations, missing_docs, unsafe_code, bare_trait_objects)]
 #![cfg_attr(feature = "long_benchmarks", feature(test))]
+#![cfg_attr(feature = "cargo-clippy", deny(clippy_pedantic))]
+#![cfg_attr(
+    feature = "cargo-clippy",
+    allow(
+          option_map_unwrap_or_else,
+          // Next cast.. lints don't give alternatives.
```

Comment:
I'd suggest replace "cast.." with \`cast_*\` or at least \`simple..\`. For me "cast.." looks like a punctuation part here.